_scicomputing/reference_training.md

```diff
@@ -57,20 +57,70 @@ Several groups on campus host weekly or monthly office hours to provide assistan
+
+- [edX](https://www.edx.org): Offers courses on [Data Analysis and Statistics](https://www.edx.org/course/subject/data-analysis-statistics) and [Bioinformatics](https://www.edx.org/learn/bioinformatics)
+  - [Rafael Irizarry](http://rafalab.github.io/) of Dana Farber has online programs available through edX:
+    - [Introduction to Data Science](https://www.edx.org/professional-certificate/harvardx-data-science)
+    - [Data Analysis for Life Sciences](https://www.edx.org/professional-certificate/harvardx-data-analysis-for-life-sciences)
+    - [Data Anaysis for Genomics](https://www.edx.org/professional-certificate/harvardx-data-analysis-for-genomics)
+  - *Generally speaking, edX courses are all free to audit for a limited period of time. Unlimited access and the ability to earn a course Certificate will require payment*
+  
+- [Coursera](https://www.coursera.org): Offers courses on [Data Science](https://www.coursera.org/browse/data-science) and [Bioinformatics](https://www.coursera.org/browse/life-sciences/bioinformatics)
+  - [R Programming](https://www.coursera.org/specializations/data-science-foundations-r): A beginner-level program has five mini-courses. It takes about 4 months to complete.
+  - [Statistics with R](https://www.coursera.org/specializations/statistics): A beginner-level program with five mini-courses. It takes about 7 months to complete.
+  - [Genomic Data Science](https://www.coursera.org/specializations/genomic-data-science): An intermediate-level program for those who are already aquainted with R. It has eight mini-courses. It takes about 6 months to complete.
+  - [Python Programming](https://www.coursera.org/specializations/data-science-python): An intermediate-level program that takes about 4 months to complete.
+  - *Coursera offers a 7-day free trial, and is a paid subscription service after*
+
+- [Udacity](https://www.udacity.com): Offers courses on [Data Science](https://www.udacity.com/courses/school-of-data-science)
+  - *Udacity is a paid subscription service*
+  - [Currently offering one month free for their Nanodegree programs.](https://blog.udacity.com/2020/03/one-month-free-on-nanodegrees.html)
+
+- [Udemy](https://www.udemy.com): Offers courses on [Data Science](https://www.udemy.com/courses/development/data-science/)
+  - *Udemy offers courses at various price points.*
+  - Keep an eye out for sales which happen regularly and can drastically reduce the cost.
+
+- [CognitiveClass.ai](https://cognitiveclass.ai/) offers a wide range of courses focused on data science, AI, and cloud computing.
+  - *All courses are free*
+
+- [The Open Source Data Science Masters](http://datasciencemasters.org): An open-source curriculum for learning data science. This is a mixed media course made up of videos, books, and slides.
+  - *Some content is free, some is paid*
+  
 - [CalTech](http://work.caltech.edu/telecourse) Learning from Data
+  - *A free YouTube series*
+
+### Interactive Coding Platforms
```

Comment:
Again, a sentence about what you mean: these still offer classes, but are work-at-your-own pace, and some have collections of practice problems too?